packages/quantic/cypress/e2e/default-1/case-classification/case-classification.cypress.ts

```diff
@@ -889,4 +897,94 @@ describe('quantic-case-classification', () => {
       });
     });
   });
+
+  describe('when the field value changes and updates-to-fetch is not undefined', async () => {
```

Comment:
`updates-to-fetch` is  a name of a variable that you are using in the component, however in the description of this test it's a bit hard to understand and not very meaningful.
I would suggest to update this with a very simple description that a person can understand without having to look at what the component does.

Reply:
good point! will change it to something more descriptive